docs/source/examples.rst

```diff
@@ -147,6 +147,49 @@ default variables. For each of the plot sets, the user needs to make a ``*_obs_v
 
 Viewing the Results
 -------------------
-Results from running all the examples can be found `here <https://portal.nersc.gov/project/e3sm/forsyth/examples/>`__.
-You can navigate to ``https://portal.nersc.gov/project/e3sm/forsyth/examples/<example directory>/viewer/`` to
+Results from running all the examples can be found `here <https://portal.nersc.gov/cfs/e3sm/forsyth/examples/>`__.
+You can navigate to ``https://portal.nersc.gov/cfs/e3sm/forsyth/examples/<example directory>/viewer/`` to
```

Comment:
Note: wait to merge until checking the correct path; see discussion on https://github.com/E3SM-Project/mache/issues/52